bin/webpack-configs.js

```diff
@@ -12,6 +12,7 @@ const WebpackRTLPlugin = require( 'webpack-rtl-plugin' );
 const chalk = require( 'chalk' );
 const { kebabCase } = require( 'lodash' );
 const CreateFileWebpack = require( 'create-file-webpack' );
+const CircularDependencyPlugin = require( 'circular-dependency-plugin' );
```

Comment:
Should probably revert this or keep it under a flag, see https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/3692

Reply:
I asked @nerrad about this. I didn't spot any obvious slow down—did you?

Reply:
If there's no slowdown it's fine to leave it.

One thing I just thought of however, does this plugin consider `.ts` and `.tsx` files in its processing? Also, the `exclude` argument seems odd to me, why are we excluding files named `a.js` (I'm assuming this was just copied from their docs).

Reply:
This runs on `npm start` only now. Other builds are not affected.